recipes/thepipe/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "thepipe" %}
+{% set version = "1.3.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/thepipe-{{ version }}.tar.gz
+  sha256: 23db69b6631633a207b42631910eeab4f67e5962088fa6b66c5b884f5227d091
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - setuptools_scm
+  run:
+    - numpy
+    - pip >=9
+    - psutil
+    - python >=3.5
+    - python-dateutil
+    - pytz
+    - setuptools-scm
```

Comment:
Are these run dependencies for sure?
- pip >=9
- setuptools-scm

Reply:
Thanks for checking! `setuptools-scm` can be skipped for the Conda package, it is only needed to determine the version number but `pip` is needed: the package queries `pip` for installed package versions and creates provenance information out of that.

Reply:
Ok, I was wrong, setuptools_scam is needed too ;)